README.md

```
@@ -1,143 +1,278 @@
-##### develop, main
+## 📖 목차
 
-- creations, updates, deletions 제한
-- PR로만 merge 가능
-- 1명 이상의 리뷰어 승인 필요
-- PR 승인 후 커밋이 발생하면 다시 승인 필요
-- status check 통과 필요
+- [**소개 (Why Pullit?)**](#-why-pullit--왜-pullit을-만들었나요)
+- [**주요 기능 (Key Features)**](#-key-features--pullit의-핵심-기능)
+- [**기술 스택 (Tech Stack)**](#️-tech-stack--pullit을-움직이는-기술들)
+- [**시스템 아키텍처 (Architecture)**](#-architecture--시스템-아키텍처)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)